website/content/docs/security/acl/acl-tokens.mdx

```diff
@@ -0,0 +1,248 @@
+---
+layout: docs
+page_title: Tokens
+description: >-
+  This topic describes access control list (ACL) tokens. Tokens are the core method of authentication in Consul.
+---
+
+# Tokens
+
+This topic describes access control list (ACL) tokens, which are the core method of authentication in Consul.
+
+## Introduction
+
+Tokens are artifacts in the ACL system used to authenticate users, services, and Consul agents. When ACLs are enabled, entities requesting access to a resource must include a token that has been linked with a policy, service identity, or node identity that grants permission to the resource. The ACL system checks the token and grants or denies access to resource based on the associated permissions.
+
+Refer to the [ACL system workflow overview](/docs/security/acl/acl-system#workflow-overview) for an information about tokens' role in the ACL system.
+
+## Creating Tokens
+
+The person responsible for administrating ACLs can use the API or CLI to create and link tokens to entities that enable permissions to resources.
+Refer to the [ACL API ](/api-docs/acl) and [ACL CLI](/commands/acl) documentation for instructions on how to create and link tokens. Tokens can also be created dynamically from trusted external system using an
+[auth method](/docs/security/acl/auth-methods).
+
+## Passing Tokens
+
+Users deploying their services into the network or performing some operations in the Consul command line or GUI will need to include the value of the token's `SecretID` attribute. The `SecretID` is often referred to as the ACL token. It is an opaque string that identifies the requester so that the ACL system can determine if access to the requested resource should be granted or denied.
+
+Tokens have several additional attributes. Refer to the (see [Token Attributes](#token-attributes)).
+
+### Service Requests
+
+Specify the value of the `SecretID` attribute with the `token` parameter when configuring your services. In the following example service configuration, the `token` has been included as a string parameter to the `redis` service:
+
+<CodeTabs>
+
+<CodeBlockConfig lineNumbers highlight="6">
+
+```hcl
+"service" = {
+  "id" = "redis"
+  "name" = "redis"
+  ...
+  "namespace" = "foo"
+  "token" = "233b604b-b92e-48c8-a253-5f11514e4b50"
+}
+```
+
+</CodeBlockConfig>
+<CodeBlockConfig lineNumbers highlight="6">
+
+```json
+{
+  "service": {
+    "id": "redis",
+    "name": "redis",
+    ...
+    "token": "233b604b-b92e-48c8-a253-5f11514e4b50",
+    "namespace": "foo"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+Refer to the [service definitions documentation](/docs/discovery/services#service-definition) for additional information.
+
+### Agent Requests
```

Comment:
Is this a request or is this configuration?

My thinking is that we should link to the configuration docs for acls, instead of duplicating the config reference here.